client/my-sites/email/email-providers-comparison/in-depth/email-provider-price.tsx

```diff
@@ -0,0 +1,30 @@
+import { useSelector } from 'react-redux';
+import { getSelectedDomain } from 'calypso/lib/domains';
+import GoogleWorkspacePrice from 'calypso/my-sites/email/email-providers-comparison/price/google-workspace';
+import ProfessionalEmailPrice from 'calypso/my-sites/email/email-providers-comparison/price/professional-email';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { EmailProviderPriceProps } from 'calypso/my-sites/email/email-providers-comparison/in-depth/types';
+import type { ReactElement } from 'react';
+
+const EmailProviderPrice = ( {
+	emailProviderSlug,
+	intervalLength,
+	selectedDomainName,
+}: EmailProviderPriceProps ): ReactElement => {
+	const selectedSite = useSelector( getSelectedSite );
+
+	const domains = useSelector( ( state ) => getDomainsBySiteId( state, selectedSite?.ID ) );
+	const domain = getSelectedDomain( {
+		domains,
+		selectedDomainName: selectedDomainName,
+	} );
+
+	if ( emailProviderSlug === 'google-workspace' ) {
```

Comment:
Let's use the appropriate constant here, instead of a literal

Reply:
I'll address that change in another pull request 👍.